posts/2021/07/hpy-0.0.2.md

```diff
@@ -0,0 +1,110 @@
+<!--
+.. title: hpy 0.0.2: First public release
+.. slug: hpy-0.0.2-first-public-release
+.. date: 2021-07-19 10:00:00 UTC
+.. author: antocuni
+.. tags:
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+hpy 0.0.2 is out! This is the first version which is officially released and
+made available on PyPI.
+
+The major highlight of this release is that it is supported by three different
+Python implementations: CPython, PyPy and GraalPython.
+
+<!--TEASER_END-->
+
+What is HPy?
+============
+
+HPy provides a new API for extending Python in C. In other words, you use
+`#include <hpy.h>` instead of `#include <Python.h>`. For more info, look at
+the
+[official documentation](https://docs.hpyproject.org/en/latest/overview.html).
+
+
+Installation
+============
+
+For CPython, you need to install it manually, using pip:
+
+```
+$ pip install hpy==0.0.2
+```
+
+!!! Note
+
+    Currently, we provide only the `sdist` (i.e., the `.tar.gz`, no binary
+    wheels). See also
+    [issue #223](https://github.com/hpyproject/hpy/issues/223), contributions
+    are welcome :).
+
+
+PyPy and GraalPython ships their own version of HPy, so no installation is
+necessary. HPy 0.0.2 will be included in the next release of both, i.e. PyPy
+x.x.x (expected on 2021-xx-xx) and GraalPython y.y.y (expected on
```

Comment:
GraalPython 21.2.0 (expected on 2021-07-20

Reply:
PyPy 7.3.6 expected in 10-2021